papers/P2988/optional_ref_wording.tex

```diff
@@ -406,6 +413,10 @@ \chapter{Impact on the standard}
 
 The proposed changes are relative to the current working draft \cite{N4910}.
 
+\chapter{Acknowledgements}
+Many thanks to all of the reviewers and authors of beman/optional26, \cite{The_Beman_Project_beman_optional26}, in particular A. Jiang, Darius Neațu, David Sankel, Eddie Nolan, Jan Kokemüller, Jeff Garland, and River. Tomasz Kamiński provided extensive support for the library wording of optional<T\&>.
```

Comment:
```suggestion
Many thanks to all of the reviewers and authors of beman/optional, \cite{The_Beman_Project_beman_optional26}, in particular A. Jiang, Darius Neațu, David Sankel, Eddie Nolan, Jan Kokemüller, Jeff Garland, and River. Tomasz Kamiński provided extensive support for the library wording of optional<T\&>.
```

It's up to you if you want to postpone the renaming from https://github.com/bemanproject/optional26/pull/102 until Austria. If already planning to merge #102 first, reminder to update this branch.

And also, thanks for acknowledge here! :D